starport/cmd/network_chain_prepare.go

```diff
@@ -21,6 +23,8 @@ func NewNetworkChainPrepare() *cobra.Command {
 	c.Flags().AddFlagSet(flagSetKeyringBackend())
 	c.Flags().AddFlagSet(flagSetHome())
 
+	var msg bytes.Buffer
+	fmt.Fprintln(&msg, "🛠️  After launch triggered, start your node by running: `{binary} --home {home} start`")
```

Comment:
Here is not the best place to put this log. Maybe you should add after the command logic. Also, you are saving the message log into the `bytes.Buffer,` but this message will never print it

Reply:
@Pantani thanks I updated the patch, is it at the correct place and with code now? if not can you help me whats exactly I am doing wrong here? thank you

Reply:
https://github.com/tendermint/starport/pull/1897#discussion_r770054010